packages/map-template/src/components/Directions/Directions.scss

```
@@ -11,6 +11,18 @@
         padding: var(--spacing-medium) 0;
     }
 
+    &__sr-heading {
+        position: absolute;
+        width: 1px;
+        height: 1px;
+        padding: 0;
+        margin: -1px;
+        overflow: hidden;
+        clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)